python/pager/flex_demod.py

```diff
@@ -24,7 +24,7 @@
 from gnuradio import blocks
 from gnuradio import filter
 from math import pi
-from pager import pager_swig
+from gnuradio.pager import pager_python
 
 
 class flex_demod(gr.hier_block2):
```

Comment:
This file still has references to `pager_swig`

Reply:
I will have a look ;)

Reply:
Yeah right, I've fixed this.